### PR TITLE
Updates Custom Weather Data Source

### PR DESCRIPTION
**Pull request by haxdds**
Updated custom data URL to Dropbox file. Updated parsing in C# to accommodate new data source.

### CODE REVIEW

File: 02 Algorithm Reference/04 Importing Custom Data/03 Creating and Reading Custom Data.html [hunk 1]
```diff
@@ -70,7 +68,7 @@
         return new Weather()
         {
             // Make sure we only get this data AFTER trading day - don't want forward bias.
-            Time = DateTime.Parse(data[0]).AddHours(20),
+            Time = DateTime.ParseExact(data[0], "yyyyMMdd", null).AddHours(20),
```
Comment (AlexCatarino) [inline 02 Algorithm Reference/04 Importing Custom Data/03 Creating and Reading Custom Data.html:71]: In the Python version, we have:
```python
weather.Time = datetime.strptime(data[0], '%Y-%m-%d') + timedelta(hours=20)
```
should it not be changed too?

Comment (haxdds) [reply 02 Algorithm Reference/04 Importing Custom Data/03 Creating and Reading Custom Data.html:71]: Aaah right. Thanks for catching that. Should be 
`datetime.strptime(data[0], '%Y%m%d') + timedelta(hours=20)`

File: 02 Algorithm Reference/04 Importing Custom Data/03 Creating and Reading Custom Data.html [hunk 2]
```diff
@@ -84,7 +82,7 @@
     ''' Weather based rebalancing'''
 
     def GetSource(self, config, date, isLive):
-        source = "https://www.wunderground.com/history/airport/{0}/{1}/1/1/CustomHistory.html?dayend=31&monthend=12&yearend={1}&format=1".format(config.Symbol, date.year);
+        source = "https://www.dropbox.com/s/m7894ciswqgsv9v/custom_weather_data.csv?dl=1"
```
Comment (jaredbroad) [inline 02 Algorithm Reference/04 Importing Custom Data/03 Creating and Reading Custom Data.html:85]: Please change the demo URL to QC's dropbox:
https://www.dropbox.com/s/8v6z949n25hyk9o/custom_weather_data.csv?dl=1

**Comment by jaredbroad** [comment]
ty @haxdds !

**Pull request merged**